### DISCUSSION TITLE
inkscape and bref

### DISCUSSION THREAD

**Discussion by hesomp** [brefphp/bref#1289] [Support]
I have a php (laravel) app deployed on Lambda via Bref. 

There is a function that I need to add that requires the inkscape library ( apt-get install inkscape). 

Could someone help me with the steps to get this installed via Bref?

**Comment by afu-dev** [comment]
Hello Hesom,

You can create a custom layer containing the Inkscape bin, publish it to your AWS account and then add it to the layer list of your function.

You should be able to download the executable here: https://inkscape.org/release/inkscape-1.2.1/source/archive/xz/

Here's the documentation from AWS to create and publish a custom layer: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html

Here's an example of a custom layer for FFmpeg: https://github.com/serverlesspub/ffmpeg-aws-lambda-layer/blob/master/Makefile

If you need more help, feel free to ask 😃
